### PR TITLE
126 enhancement suggestion turn off cell type clustering if bulk tissue

### PR DESCRIPTION
**Pull request by ejh243**
# Description

This pull request will make the cell type clustering check optional. To achieve this I have merged the two QC reports into one, with the second one now an optional chunk based on the `ctCheck` variable in the R config file. Had to move some other code sections around and reorder the commands in the job submission script. 

## Issue ticket number

This pull request is to address issue: #126.

## Type of pull request

- [ ] Bug fix
- [X] New feature/enhancement
- [X] Code refactor
- [ ] Documentation update

## Checklist

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have tested my code to check that it is functional
- [ ] I have used linters to check for common sources of errors
- [ ] I have implemented fail safes in my code to account for edge cases
- [ ] I have made the corresponding changes to the documentation

### CODE REVIEW

**Comment by ew367** [comment]
@ejh243 Has this code been tested on any of the devData datasets yet? Is there any example output for the new combined QC report anywhere?

**Comment by ejh243** [comment]
YEs I have tested it on my local copy of devData. I will send you the report output

File: array/DNAm/preprocessing/rmarkdownChild/ctCheck.rmd [hunk 1]
```diff
@@ -1,50 +1,18 @@
...
-    toc: true
-    toc_float: true
-    number_sections: true
----
-```{r global_options, include=FALSE}
-knitr::opts_chunk$set(fig.align = "center", warning=FALSE, message=FALSE, echo = FALSE, cache = FALSE)
-```
 
 ```{r setup, include=FALSE}
-args<-commandArgs(trailingOnly = TRUE)
```
Comment (ew367) [inline array/DNAm/preprocessing/rmarkdownChild/ctCheck.rmd:15]: first chunk name 'setup' is duplicate of QC.rmd chunk name so causes error/crash

File: array/DNAm/preprocessing/QC.rmd
Comment (ew367) [inline array/DNAm/preprocessing/QC.rmd:1]: The example config file in this PR has "tissueType<-"FANS sorted post-mortem brain tissue"
It needs to be "blood" or "brain" as it is used by cetygo to call the correct reference panel

**Review by ew367 — CHANGES_REQUESTED** [review]

**Comment by ejh243** [comment]
@ew367 all changes made and tested with both ctCheck = to TRUE and FALSE, can you re-review?

**Review by ew367 — APPROVED** [review]
re-tested in devData with ctCheck=T and ctCheck=F and works fine on both

**Pull request merged**